engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -88,36 +76,16 @@ public boolean equals(Object obj) {
 
     @Override
     public void update() {
-        task.setFboId(fboManager.get(fboName).fboId);
+        fboId  = fboManager.get(fboName).fboId;
     }
 
     @Override
     public String toString() { // TODO: used for logging purposes at the moment, investigate different methods
-        return String.format("%30s: %s", this.getClass().getSimpleName(), fboName);
+        return String.format("%30s: %s (fboId:%s)", this.getClass().getSimpleName(), fboName, fboId);
     }
 
-    private final class BindFBOTask implements RenderPipelineTask {
-
-        private int fboId;
-        private final ResourceUrn fboName;
-
-        private BindFBOTask(int fboId, ResourceUrn fboName) {
-            this.fboId = fboId;
-            this.fboName = fboName;
-        }
-
-        @Override
-        public void execute() {
-            glBindFramebufferEXT(GL_FRAMEBUFFER_EXT, fboId);
-        }
-
-        private void setFboId(int fboId) {
-            this.fboId = fboId;
-        }
-
-        @Override
-        public String toString() {
-            return String.format("%30s: %s (fboId:%s)", this.getClass().getSimpleName(), fboName, fboId);
-        }
+    @Override
+    public void process() {
+        glBindFramebufferEXT(GL_FRAMEBUFFER_EXT, fboId);
```

Comment:
Bummer. GL_DRAW_FRAMEBUFFER is only available from OpenGL 3.0. :disappointed: Never mind then. Let's just add:

// TODO: change the target argument to GL_DRAW_FRAMEBUFFER when we switch to OpenGL 3.0 and beyond.